src/Columns/ColumnDecimal.cpp

```diff
@@ -386,11 +386,11 @@ ColumnPtr ColumnDecimal<T>::compress() const
 
     const size_t compressed_size = compressed->size();
     return ColumnCompressed::create(data_size, compressed_size,
-        [compressed = std::move(compressed), column_size = data_size, scale = this->scale]
+        [c_compressed = std::move(compressed), column_size = data_size, c_scale = this->scale]
```

Comment:
it's hard to diff, I do not think we have a script to support on this.
basically, when encountering a warning, we intends to make the code change same as clickhouse way. like replace 
```suggestion
        [my_compressed = std::move(compressed), column_size = data_size, my_scale = this->scale]
```

https://github.com/ClickHouse/ClickHouse/blob/daf1ca06253ff4370622aa45befc00a2af19e1d6/src/Columns/ColumnDecimal.cpp#L486-L492